test/runtime/samples/input-crazy-value-attribute/_config.js

```diff
@@ -0,0 +1,10 @@
+export default {
+	skip_if_hydrate_from_ssr: true,
+	skip_if_hydrate: true,
+	skip_if_ssr: true,
+
+	test({ assert, target }) {
+		const input = target.querySelector('input');
+		assert.equal(input.value.length > 100_000, true);
```

Comment:
numeric separators were added in Node 12.5 and we support older versions, which is why the CI is failing
```suggestion
		assert.equal(input.value.length > 100000, true);
```

Reply:
Ahh, makes sense! I'll resolve this, and also increase the value because the value is much longer than 100 000 characters